### DISCUSSION TITLE
.rbxl file?

### DISCUSSION THREAD

**Discussion by markfrappy** [littensy/slither#183] [General]
Hello,
Any way of posting a .rbxl file?  
Or tips on how I compile this into a .rbxl?

Thanks

**Comment by littensy** [comment]
Hi! You can build the project by cloning the repository and running the following commands:
```bash
# Install dependencies
yarn
aftman install

# Build project
yarn build:dev
```
Note that further configuration is needed on your end to get the store and badges to work. I'd recommend looking at [Mantle's documentation](https://mantledeploy.vercel.app/).

**Comment by markfrappy** [comment]
Thanks for replying.  To clone it, what steps do I do for that.... sorry I am new to using Github.  Also is there a direct way of msg you, like discord?

**Reply by littensy** [reply]
You can contact me **@littensy** on Discord – clone a repository by running `git clone https://github.com/littensy/slither.git` in the terminal at the location you wish to clone it in.

**Reply by markfrappy** [reply]
ok I sent you a friend discord invite
